### PR TITLE
315484 Create MVC with ASP .NET Core Sample for Iframe Full Server Based Embedding

### PR DESCRIPTION
**Pull request by anuabarnabsf4019**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,78 @@
-# Bold BI
+# Bold BI Embedded Sample in .NET Core MVC
```
Comment (vinothkumar1732) [inline README.md:1]: This is common readme.md file for entire repo. You have to maintain your Readme.md file inside your folder alone. Revert this changes.

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample.sln [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample.sln:2]: **EmbeddedShell** folder is not needed. Previous structure is not correct.

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Controllers/HomeController.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        {
+            ViewBag.IsAuthenticated = HttpContext.User.Identity.IsAuthenticated;
+            ViewBag.ReturnURL = Request.Query["returnURL"].ToString();
+            return View();
+        }
+
+        public ActionResult Logout()
+        {
+            HttpContext.SignOutAsync("Cookies");
+            var url = "http://localhost:58295/oauth/logout?redirect_uri=localhost:44383/Home/Loginpage";// Here localhost with port number 53623 refers the Bold BI server URL
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Controllers/HomeController.cs:84]: Can't we use maintain the Application Base URL and Bold BI Server URL in appsettings.json as other values?

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Models/TokenHelper.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            var claims = new[] {
+        new Claim(JWTSSOClaims.Upn, new System.Net.Mail.MailAddress(userInfo.Email).User),
+        new Claim(JWTSSOClaims.Email, userInfo.Email),
+        new Claim(JWTSSOClaims.FirstName, new System.Net.Mail.MailAddress(userInfo.Email).User),
+        new Claim(JWTSSOClaims.LastName, new System.Net.Mail.MailAddress(userInfo.Email).User),
+        new Claim(JWTSSOClaims.Sub, _configuration["user:userid"].ToLower())
+            };
+
+            var token = new JwtSecurityToken(claims: claims,
+                expires: DateTime.Now.AddMinutes(120),
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Models/TokenHelper.cs:35]: Customize the minutes with appSettings.json

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Program.cs [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        services.AddControllersWithViews();
+
+        services.AddAuthentication(options =>
+        {
+            options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+        })
+        .AddCookie(options =>
+        {
+            options.Cookie.Name = "YourCookieName"; //Use any name
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Program.cs:46]: Mention proper cookie name.

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Views/_ViewImports.cshtml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+﻿@using IframeFullServerSample
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Views/_ViewImports.cshtml:1]: Check all the Views are needed for this sample? If not, please remove.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
+# Bold BI Embedded Sample in .NET Core MVC
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:1]: Change the Heading suits for this sample.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+This project was created using ASP.NET Core MVC 6. The application aims to demonstrate how to render Bold BI server with JWT in Iframe Embedding.
+
+## Bold BI Server view
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:5]: Bold BI Server Embedding view

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anuabarnabsf4019) [reply README.md]: Reverted the Readme.md file changes

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Controllers/HomeController.cs (hunk 3 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Controllers/HomeController.cs]: Maintained Bold BI Server URL from appsettings.json file

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Program.cs (hunk 5 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Program.cs]: Mentioned proper cookie name

File: Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Views/_ViewImports.cshtml (hunk 6 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Views/_ViewImports.cshtml]: Checked and removed which are not needed

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 7 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Changed heading

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 8 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+![Bold BI Server View](https://github.com/bold-bi/embedded-bi-samples/assets/129487075/23c26113-126e-4e09-b9b8-5faf8d554b09)
+
+## Requirements/Prerequisites
+
+ * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Checked in Visual Studio Code and application worked properly

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+ * [.NET Framework 6](https://dotnet.microsoft.com/download/dotnet-core)
+
+#### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, Safari and MAC.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Removed from supported browsers

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+ ## Configuration
+ 
+ * Please ensure that you have enabled JWT Authentication on UMS page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/multi-tenancy/site-administration/authentication/json-web-token/#steps-to-configure-jwt-in-bold-bi) to enable it.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Added UMS mock link

[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Updated steps in README.md file

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+ 
+    ![JWT Authentication](https://github.com/bold-bi/embedded-bi-samples/assets/129487075/99381eeb-5b82-4a84-843f-56417efb782e)
+
+ * Enter `Remote Login URL` and `Remote Logout URL` as they appear in the image.  Copy the `Signing Key` from the JWT Authentication page and paste it into the `jwt:signingkey` value in the `appsettings.json` file.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+
+ * Open the UMS Site Settings page in Bold BI Server. Within `Authentication`, select the `General` tab, enable `Enable Default Authentication`, and `save` the changes.
+ 
+    ![Default Authentication](https://github.com/bold-bi/embedded-bi-samples/assets/129487075/e4fd3ec2-f54b-4a0b-a7ac-ea2e70e89b62)
+    
+ * Search for the `config.xml` file in the `Configuration` directory and change the value of the `<EnableSameTabLinkTarget>` tag from `false` to `true` in order to render the URL within the same application."
+    
+    ![Configuration](https://github.com/bold-bi/embedded-bi-samples/assets/129487075/cd177ca7-2218-47c4-a05e-2a60a545a1a0)
+
+ * Open the Administration Site Settings page in Bold BI Server. Under `Authentication`, select the `General` tab and enable `Enable Default Authentication`, then `save` the changes.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Added mock link of Bold BI Server URL

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/97a6e08b-328a-42fb-a323-1b3784420960)
+    
+ * Copy the `Signing Key` from the JWT Authentication page and paste it into the `jwt:signingkey` value in the `appsettings.json` file.
+ 
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/37f439d6-3f58-43b3-807f-b90aeb3e2fbe)
+
+ * Open the UMS Site Settings page in `Bold BI Server`. Within `Authentication`, select the `General` tab, enable `Enable Default Authentication`, and `save` the changes.
+ 
+    ![Default Authentication](https://github.com/boldbi/samples/assets/129487075/9de1893d-52f0-40f7-b246-35169e97012a)
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:36]: Change the image with Marking Authentication section and JWT SSO selected as default authentication.

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]
As we discussed in the call, please revisit all the points, links and images as well.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 15]
```diff
@@ -0,0 +1,88 @@
+# Bold BI Full Server Based Embedding in ASP .NET Core MVC using IFrame
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+
+This project was created using ASP.NET Core MVC. The application aims to demonstrate how to render Bold BI server with JWT in Iframe Embedding.
+
+## Bold BI Server Embedding view
+
+![Bold BI Server View](https://github.com/boldbi/samples/assets/129487075/37e0af6a-bd96-478c-ae91-5948840f257d)
+
+## Requirements/Prerequisites
+
+ * [Visual Studio Code](https://code.visualstudio.com/download)
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:11]: Updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 17]
```diff
@@ -0,0 +1,88 @@
...
+ * [.NET Framework](https://dotnet.microsoft.com/download/dotnet-core)
+
+ > **NOTE:** .NET 6.0 and 7.0 are supported
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Created using .Net 6 and updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+
+#### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox and Safari.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:16]: Not ensured in mac

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 19]
```diff
@@ -0,0 +1,88 @@
...
+
+ ## Configuration
+ 
+ * Please configure the JWT Authentication in you UMS page based on your application Login and Logout URL. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/multi-tenancy/site-administration/authentication/json-web-token/#steps-to-configure-jwt-in-bold-bi) to enable it. (ex: http://localhost:50000/ums/administration)
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/0dc156e9-4d18-4ebf-b671-a72e045c70a6)
+
+ * As the application based url is localhost, we are setting the `Remote Login URL` and `Remote Logout URL` as below.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: Updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 21]
```diff
@@ -0,0 +1,88 @@
...
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/97a6e08b-328a-42fb-a323-1b3784420960)
+    
+ * Copy the `Signing Key` from the JWT Authentication page and paste it into the `jwt:signingkey` value in the `appsettings.json` file.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:28]: updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 22]
```diff
@@ -0,0 +1,88 @@
...
+ 
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/37f439d6-3f58-43b3-807f-b90aeb3e2fbe)
+
+ * Open the UMS Site Settings page in `Bold BI Server`. Within `Authentication`, select the `General` tab, enable `Enable Default Authentication`, and `save` the changes.
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 14 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md]: changed the image

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 23]
```diff
@@ -0,0 +1,88 @@
...
+
+ * Open the UMS Site Settings page in Bold BI Server.  Under the `Authentication`, select the `General` tab and enable `Enable Default Authentication`.  In the `Default Authentication Provider` dropdown, select `JWT SSO` and `Save` the changes.
+ 
+    ![Default Authentication](https://github.com/boldbi/samples/assets/129487075/a81894ac-c147-41df-8c97-ed9928d16953)
+    
+ * Click on `Configuration` in UMS Settings, then select the `config.xml` file from the dropdown.  Change the value of the `<EnableSameTabLinkTarget>` tag from `false` to `true` in order to render the URL within the same application and `Save` the changes.
+    
+    ![Configuration](https://github.com/boldbi/samples/assets/129487075/b3d65c3c-d470-4f53-9983-a35291bb6bfe)
+
+ * Open the Administration Site Settings page in Bold BI Server. Under `Authentication`, select the `General` tab and enable `Enable Default Authentication`, then `Save` the changes. (ex: http://localhost:50000/bi/site/site1/administration)
```
[the remark this answers is not in the document]
Comment (vinothkumar1732) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:42]: In this step, we didn't set any default authentication in dropdown. Whether this is needed? Please ensure the working case with disabled the Default Authentication.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 24]
```diff
@@ -0,0 +1,88 @@
...
+    ![Admin Authentication](https://github.com/boldbi/samples/assets/129487075/763ae9a6-4448-40c3-9897-afb28db745ed)
+
+ * In the application, change the `jwt:boldbiserverurl` value in the `appsettings.json` file to the URL of our Bold BI server.
+  
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/efb08c80-ffcc-453e-b216-0949b06b9126)
+
+    
+ ## Run a Sample Using Command Prompt 
+    
+  1. Open the command prompt and navigate to the specified file [location](https://github.com/boldbi/samples/tree/master/Scenario%20Based%20Samples/Iframe%20Full%20Server%20Embedding/IframeFullServer) where the project is located.
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:53]: Have you ensured this sample from command prompt?

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 25]
```diff
@@ -0,0 +1,88 @@
...
+
+  2. Execute the command `dotnet restore` to restore the necessary packages. Once the packages have been successfully restored, use the `dotnet build` command to build the project.
+  
+  3. Finally, run the application using the command `dotnet run`.  It will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
+
+ ## Developer IDE
+
+  * [Visual studio code](https://code.visualstudio.com/download)
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:61]: Visual Studio Code

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 26]
```diff
@@ -0,0 +1,88 @@
...
+  
+### Run a Sample Using Visual Studio Code
+ 
+  * Open the folder `IframeFullServer` in Visual Studio Code.
+  
+  * Open the terminal in Visual Studio Code and execute the command `dotnet restore` to restore the required dependencies.
+ 
+  * Build your .NET project by executing the `dotnet build` command in the terminal.
+ 
+  * To run the application, use the command `dotnet run` in the terminal. After executing the command, the application has started and it will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:71]: (e.g., http://localhost:4000) 

- URL port 4 digits only.

Comment (vinothkumar1732) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:71]: Copy the URL and paste it into your web browser.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 27]
```diff
@@ -0,0 +1,88 @@
...
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/efb08c80-ffcc-453e-b216-0949b06b9126)
+
+    
+ ## Run a Sample Using Command Prompt 
+    
+  1. Open the command prompt and navigate to the specified file [location](https://github.com/boldbi/samples/tree/master/Scenario%20Based%20Samples/Iframe%20Full%20Server%20Embedding/IframeFullServer) where the project is located.
+
+  2. Execute the command `dotnet restore` to restore the necessary packages. Once the packages have been successfully restored, use the `dotnet build` command to build the project.
+  
+  3. Finally, run the application using the command `dotnet run`.  It will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:57]: Copy the URL and paste it into your web browser.

(e.g., http://localhost:4000/)

    URL port 4 digits only.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 28]
```diff
@@ -0,0 +1,88 @@
...
+  
+  * Open the terminal in Visual Studio Code and execute the command `dotnet restore` to restore the required dependencies.
+ 
+  * Build your .NET project by executing the `dotnet build` command in the terminal.
+ 
+  * To run the application, use the command `dotnet run` in the terminal. After executing the command, the application has started and it will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
+    
+    ![Bold BI Server image](https://github.com/boldbi/samples/assets/129487075/37e0af6a-bd96-478c-ae91-5948840f257d)
+
+    > **NOTE:** To mitigate issues related to NuGet packages, run the following command in terminal `dotnet add package Microsoft.IdentityModel.Tokens` and `dotnet add package System.IdentityModel.Tokens.Jwt`
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Iframe Full Server Embedding/README.md:75]: Note should not parallel to image. Must be like below.
image

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]
Review concerns added. Resolve and submit again.

File: Scenario Based Samples/Iframe Full Server Embedding/README.md [hunk 29]
```diff
@@ -0,0 +1,88 @@
...
+
+ ## Configuration
+ 
+ * Please configure the JWT Authentication on your UMS page (ex: http://localhost:50000/ums/administration).  To do this, navigate to the UMS Site Settings page on the Bold BI Server. From there, choose `Authentication` and click on the `JWT` tab.  Enable the `Enable JWT`. If it's not, please refer to the following image or detailed [instructions](https://help.boldbi.com/multi-tenancy/site-administration/authentication/json-web-token/#steps-to-configure-jwt-in-bold-bi) on how to enable it. 
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/4e90e36d-2eeb-408b-b215-ca48160f5719)
+
+ * Provide the following details in the `JWT`.  As the application based url is localhost, we are setting the `Remote Login URL` and `Remote Logout URL` as below and `Save` the changes.
+ 
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/129a52df-d7e9-4b82-8ca5-7300e22ea4e3)
```
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:26]: changed the image

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 25 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:61]: updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 26 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:71]: updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 27 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:57]: updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 28 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:75]: updated

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 24 above)
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:53]: Yes checked and working properly

File: Scenario Based Samples/Iframe Full Server Embedding/README.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (anuabarnabsf4019) [reply Scenario Based Samples/Iframe Full Server Embedding/README.md:42]: Checked if its disabled also application rendered properly so I removed this point

**Review by vinothkumar1732 — APPROVED** [review]
